torchaudio/prototype/functional/functional.py

```diff
@@ -0,0 +1,83 @@
+import torch
+
+
+def _check_convolve_inputs(x: torch.Tensor, y: torch.Tensor) -> None:
+    if x.shape[:-1] != y.shape[:-1]:
+        raise ValueError(f"Leading dimensions of x and y don't match (got {x.shape} and {y.shape}).")
+
+    if x.dim() < 2:
```

Comment:
wondering why the method doesn't support 1D Tensor?

Reply:
thanks — fixed